test/utils/InstanceTest.sol

```diff
@@ -244,8 +244,7 @@ abstract contract InstanceTest is HyperdriveTest {
         }
     }
 
-    /// @dev Deploys the Hyperdrive Factory contract and
-    ///      sets the default pool configuration.
```

Comment:
This comment was shorter to stay within 80 chars. We'll want to do this everywhere to stay consistent.

Reply:
Is the style requirement only comment part or entire line (including natspec)? Do you use a plugin to automatically do this for you?

Reply:
It's all of it. I don't have a plugin do it, but I do have a marker in my editor that tells me where 80 characters is. A plugin would be great.